packages/neon_framework/lib/src/platform/web.dart

```diff
@@ -39,8 +40,10 @@ class WebNeonPlatform implements NeonPlatform {
   bool get canUsePaths => false;
 
   @override
-  FutureOr<void> init() {
+  FutureOr<void> init() async {
```

Comment:
```suggestion
  Future<void> init() async {
```

There is no way this will complete synchronously